src/types.ts

```diff
@@ -20,19 +20,30 @@ import { IMocks } from 'graphql-tools'
 import { IMiddleware as IFieldMiddleware } from 'graphql-middleware'
 
 export type Omit<T, K> = Pick<T, Exclude<keyof T, K>>
-export interface IResolvers {
-  [key: string]: (() => any) | IResolverObject | GraphQLScalarType
+
+export type IEnumResolver = { [key: string]: string | number };
+export interface IResolvers<TSource = any, TContext = any> {
```

Comment:
I think `= unknown` would be preferable in TypeScript v3.

Reply:
`unknown` and `any` have functional differences. When using unknown you are forced to either not use the variable or check its type.

I personally prefer `unknown`, but I don't think it is something this library would want to enforce on users as it can break existing code.

The following works with `any`, but not if changed to `unknown`

```ts
const resolver: GraphQLFieldResolver<unknown, unknown> = (source, args) => {
    // Type 'unknown' is not assignable to type 'T'.
    const foo: T = source;
    const bar: T = args;

    // Object is of type 'unknown'.
    const foobar = source.prop;
    const barfoo = args.id;
};
```